src/stylesheets/screens/survey-detail.scss

```diff
@@ -0,0 +1,55 @@
+.survey-detail {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-left {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__description {
+    @include details-container();
+    position: relative;
+
+    flex: 1;
+  }
+
+  &__image {
+    width: 100%;
+    height: rem(302px);
+    margin-top: rem(55px);
+
+    border-radius: rem(12px);
+  }
+
+  &__title {
+    margin-top: rem(20px);
```

Comment:
@rafayet-monon Should we change this to `rem(32px)`?
![Screen Shot 2563-11-24 at 17 34 53](https://user-images.githubusercontent.com/1772999/100083207-b48e1e80-2e7b-11eb-9e0f-0b630e58a3b9.png)